rune/libenclave/attestation/sgx/ias/api.go

```diff
@@ -1,7 +1,7 @@
 package ias
 
-const (
-	apiVersion = 3
+var (
+	apiVersion uint64
```

Comment:
You can still keep this global variable and assign it to v4 as currently supported version.